jep/0000/README.adoc

```diff
@@ -0,0 +1,358 @@
+= JEP-0000: End Java 8 support in Jenkins
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| End Java 8 support in Jenkins
+
+| Sponsor
+| link:https://github.com/MarkEWaite[Mark Waite]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Process
+
+| Created
+| 2022-01-28
+
+| BDFL-Delegate
+| TBD
+
+| Pull requests
+a|
+
+- link:https://github.com/jenkinsci/jenkins/pull/6083[PR 6083] - Drop Java 8 from Jenkins core
+- link:https://github.com/jenkinsci/jenkins/pull/6086[PR 6086] - Switch Jenkins acceptance test harness in core to Java 11
+- link:https://github.com/jenkinsci/jenkins/pull/6092[PR 6092] - Add administrative monitor to remind of upcoming end of Java 8 support
+- link:https://github.com/jenkinsci/acceptance-test-harness/pull/726[PR 726] - Drop Java 8 from acceptance test harness
+
+| Discussions-To
+| link:https://groups.google.com/g/jenkinsci-dev[Jenkins Developer Mailing List]
+
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+== Abstract
+
+Java 11 was released on September 25, 2018.
+It is an LTS version with a long support timeline.
+Java 8 was released in March 2014.
+The link:https://www.oracle.com/java/technologies/java-se-support-roadmap.html[Oracle Java SE Support Roadmap] states that premier support for Java 8 ends in March 2022.
```

Comment:
yeah that's what i'm worried about putting this in a reasoning, it will give people who are against this an argument, well my vendor is supporting it till 2030...

Reply:
If there is a legitimate argument then it should be discussed in a section in **Reasoning**, with an explanation of why we chose this approach nonetheless.

Reply:
Points well taken.  Removed in 5dec80af82a1e679f587a3139303fcf0598de9d9